massband/diffusion/kinisi_diffusion.py

```
@@ -1,239 +1,266 @@
-                displacement_unit="Å$^2$",
-                value_label="D",
-                value_unit="cm$^2$s$^{-1}$",
-                msd_title=f"{data.structure} MSD with std",
-                msd_filename=f"{data.structure}_msd_std.png",
-                ci_title=f"{data.structure} MSD credible intervals",
-                ci_filename=f"{data.structure}_credible_intervals.png",
-                hist_title=f"{data.structure} Diffusion Histogram",
-                hist_filename=f"{data.structure}_hist.png",
-                hist_label="MAP (D_n)",
+        # Add σ-based intervals as vertical lines
+        for i, (ci, s) in enumerate(zip(credible_intervals, sigmas)):
```

Ruff (0.12.2)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)